Modules/mathmodule.c

```diff
@@ -2496,6 +2496,146 @@ math_isclose_impl(PyObject *module, double a, double b, double rel_tol,
 }
 
 
+/*[clinic input]
+math.comb
+
+    a: object
+
+    b: object
+
+
+Return the binomial coefficient indexed by the pair of integers n >= k >= 0.
```

Comment:
How "a" and "b" relate to n and k?

Reply:
Didn't notice the ambiguity, replacing them with a as n and b as k